numpy/core/feature_detection_cmath.h

```diff
@@ -31,6 +31,8 @@ cfloat ctanhf(cfloat);
 float crealf(cfloat);
 float cimagf(cfloat);
 cfloat conjf(cfloat);
+float cabsf(cfloat);
+float cargf(cfloat);
```

Comment:
The changes in this file were probably intended to be reverted too, or did you leave them in on purpose?

Reply:
Correct, reverting now.